libpod/container_internal.go

```diff
@@ -552,6 +554,11 @@ func (c *Container) reinit(ctx context.Context) error {
 	if err := c.runtime.ociRuntime.deleteContainer(c); err != nil {
```

Comment:
@wking It might be worth making a wrapper for this that calls `deleteContainer()` and then `postDeleteHooks()`, given that we want to always call postrun hooks deleting the container in runc.